frontend_app_admin_workspace_user/i18next.scanner/en/translation.json

```diff
@@ -36,14 +37,21 @@
   "Password": "Password",
   "Create the user": "Create the user",
   "Create a user": "Create a user",
-  "Please type a name, an email, a password and select a profile": "Please type a name, an email, a password and select a profile",
+  "Please enter a name, a password, select a profile, and at least one between username or email": "Please enter a name, a password, select a profile, and at least one between username or email",
   "Please set a password": "Please set a password",
   "Full name": "Full name",
   "Full name must be at least 3 characters": "Full name must be at least 3 characters",
+  "Username must be at least 3 characters": "Username must be at least 3 characters",
```

Comment:
Use a variable `minimumCharactersUsername`.

See this diff, there is the same thing there: https://github.com/tracim/tracim/pull/2992/files#diff-0df6aac939387981abcde167323bb25eR250

Reply:
Changed